zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -226,6 +226,9 @@ public void onMessage(NotebookSocket conn, String msg) {
           case LIST_UPDATE_NOTEBOOK_JOBS:
             unicastUpdateNotebookJobInfo(conn, messagereceived);
             break;
+          case EDITOR_SETTING:
+            getEditorSetting(conn, notebook, messagereceived);
```

Comment:
As far as I understood we introduce this new message type to current WS api in order just to pass this information to the front-end, right?

I wonder if there is another, simpler way to do it? As we are not changing it from the front, there is much less value in broadcasting it though WS (that's what we use WS mostly for), so it makes me think that there must be a simpler way to just pass this information to the front..or do I miss something here?

What do you think?